client/src/components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "../styles/Footer.css";
+import CYFLogo from "../assets/CYF.jpeg";
+import { Divider, Grid, Image, Segment } from "semantic-ui-react";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <Grid divided columns={3}>
+        <Grid.Column>
+          {" "}
+          <div className="team-container">
+            <p>Min-Imece-Team</p>
```

Comment:
```suggestion
            <h3>Min-Imece-Team</h3>
```
It looks like this is a heading for all the team members — let's make it a heading